android/java/res/drawable/brave_sync_toggle_off_btn.xml

```diff
@@ -7,14 +7,5 @@
 
 <shape xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
this drawable is only being used for the transparent bg ? can we use transparent bg instead ?

Reply:
yes, indeed.
fixed at https://github.com/brave/brave-core/pull/23051/commits/3c79b2b8cdbd58c4ff1dda3ea63e4529942b74a4